website/blog/2022-01-06-open-sourcing-sorbet-vscode.md

```diff
@@ -0,0 +1,108 @@
+---
+id: open-sourcing-sorbet-vscode
+title: Open-sourcing the Sorbet VS Code Extension
+---
+
+We’re excited to announce that Stripe’s VS Code extension for Sorbet is now
+[open source](https://github.com/sorbet/sorbet/tree/master/vscode_extension).
+We’ve designed Sorbet to be used in editors from Day 1—For the past two years,
+Sorbet has exposed a flag (`--lsp`) that starts Sorbet in
+[Language Server Protocol](https://microsoft.github.io/language-server-protocol/)
+(LSP) mode. In this mode, Sorbet can respond to many LSP requests, like Go To
+Definition, Find All References, Autocomplete, and more.
+
+With this release, we’re making it even easier to take advantage of these LSP
+features when working with Sorbet in VS Code. We hope that this extension takes
+some of the guess work out of how to configure and use Sorbet’s LSP mode.
+
+<!--truncate-->
+
+Today’s release includes:
+
+- The
+  [pre-built extension](https://marketplace.visualstudio.com/items?itemName=sorbet.sorbet-vscode-extension)
+  on the Visual Studio Marketplace
+- The [source code](https://github.com/sorbet/sorbet/issues/new/choose) for the
```

Comment:
I believe this should rather link to https://github.com/sorbet/sorbet or https://github.com/sorbet/sorbet/tree/master/vscode_extension - I got pretty confused by being greeted with the "New Issue" view when trying to take a look at the code :)